### ISSUE TITLE
Jetpack Focus - Accessibility - Large Text Size not applying correctly to Badge on Activity Detail

### ISSUE THREAD

**Issue by ovitrif** [wordpress-mobile/WordPress-iOS#19125]
### Expected behavior
The Jetpack Powered badge text scales nicely with the large font size set on the iPhone.

### Actual behavior
The Jetpack Powered badge font size is not scaled while there is an increasing inner padding to the green area.

### Steps to reproduce the behavior
1. Go to Settings → Accessibility
2. Tap `Display & Text Size` then tap `Larger Text`
3. Slide the text size control to the right edge
4. Open WordPress app
5. Tap `Menu` then tap `Activity Log`
6. Tap any activity item
   You need one activity item, if missing one quickly change the site name to generate an activity log item.
7. **Verify** the text on the Jetpack Powered Badge is not scaled to match the large text size & it has an extended right padding.

🗣 **Remark** The non-scaling text size might be not that much of an issue, but the inner padding could do great with a fix.

| Expected | Actual |
| --- | --- |
| <img width="314" src="https://user-images.githubusercontent.com/4588074/181740724-2f5c09cd-dfa7-4cb9-b68c-f9184579537f.png"> | <img width="314" src="https://user-images.githubusercontent.com/4588074/181739319-a9528fad-b4d5-4919-88cb-de505931f9ed.jpeg"> |
